pkg/cli/server/server.go

```diff
@@ -82,7 +82,11 @@ func run(app *cli.Context, cfg *cmds.Server) error {
 	if err != nil {
 		return errors.Wrapf(err, "Invalid CIDR %s: %v", cfg.ClusterCIDR, err)
 	}
-
+	_, serverConfig.ControlConfig.ServiceIPRange, err = net2.ParseCIDR(cfg.ServiceCIDR)
+	if err != nil {
+		return errors.Wrapf(err, "Invalid CIDR %s: %v", cfg.ServiceCIDR, err)
+	}
+	serverConfig.ControlConfig.ClusterDNS = net2.ParseIP(cfg.ClusterDNS)
```

Comment:
I'd prefer that by default the cluster-dns is blank "" and if it's blank take the service-cidr and add 10.  With that approach if somebody changes the service cidr they also don't have to remember to change the cluster-dns and it will assume a sane value.  Then if they really really care for the cluster-dns to be a specific IP then they can also pass `--cluster-dns`.

Reply:
Makes perfect sense, fixed.